src/scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/util/HealthUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.cloudfoundry.autoscaler.scheduler.util;
+
+import org.cloudfoundry.autoscaler.scheduler.conf.HealthServerConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HealthUtils {
+
+  @Autowired static HealthServerConfiguration healthServerConfig;
+
+  private HealthUtils() {}
+
+  public static String livenessUrl() {
+    return "http://localhost:" + healthServerConfig.getPort() + "/health/liveness";
+  }
+
+  public static String prometheusMetricsUrl() {
+    return "http://localhost:" + healthServerConfig.getPort() + "/health/prometheus";
+  }
```

Comment:
Why using `String` instead of the std-lib type [URL](<https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/net/URL.html>)?